### DISCUSSION TITLE
Cosy 6 Heatpump

### DISCUSSION THREAD

**Discussion by D3RON** [BottlecapDave/HomeAssistant-OctopusEnergy#1044] [Ideas]
Hi, does anyone know if the Octopus Cosy 6 heat pump has api data?

**Comment by BottlecapDave** [comment]
Hello. There are various APIs relating to heat pumps. Please upvote https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/972 if you're interested in this, and please provide requested information as per https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/972#issuecomment-2411758094
